proposals/2020-10-28-Experimental.md

```diff
@@ -113,12 +113,13 @@ This kind of requirement is handled for TypeScript libraries by [api-extractor](
 ### Unanswered Questions
 
 - If an operation is not marked as experimental, but it is using a schema which is (i.e. as its request object), then it is implicitly also unstable. Would this usage be considered invalid?
-- Should `experimental` and `deprecated` be mutually exclusive?
 
 ## Backwards compatibility
 
 The `experimental` field would default to false, meaning existing behaviour is preserved, and the new field is only used on an opt-in basis.
 
+`experimental` can coexist with `deprecated` - an operation, parameter or schema can be both experimental and deprecated, having never gotten to a stable point before being deprecated.
```

Comment:
Not sure whether I'd bother with deprecating an experimental feature, the Motivation section states that experimental aspects can

> just get removed

and the Proposed solution section repeats that it 

> may change or be removed without a major version bump

What would it mean to deprecate an experimental feature as opposed to not removing it? That it will still work in this major version and disappear with the next major version bump?